app/src/main/java/com/sopt/now/homeFragment/HomeFragment.kt

```diff
@@ -0,0 +1,44 @@
+package com.sopt.now.homeFragment
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.fragment.app.Fragment
+import androidx.fragment.app.viewModels
+import androidx.recyclerview.widget.LinearLayoutManager
+import com.sopt.now.databinding.FragmentHomeBinding
+
+class HomeFragment : Fragment() {
+
+    private var _binding: FragmentHomeBinding? = null
+    private val binding
+        get() = requireNotNull(_binding) {
+            "바인딩 객체 생성 해라"
+        }
+    private val viewModel by viewModels<HomeViewModel>()
+
+    override fun onCreateView(
+        inflater: LayoutInflater,
+        container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View {
+        _binding = FragmentHomeBinding.inflate(inflater, container, false)
+        return binding.root
+    }
+
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+        val homeProfiledAdapter = HomeProfileAdapter()
+        binding.rvHomeFriendList.run {
+            adapter = homeProfiledAdapter
+            layoutManager = LinearLayoutManager(requireContext())
+        }
+        homeProfiledAdapter.setProfileList(viewModel.mockMyProfile, viewModel.mockFriendProfileList)
+    }
+
+    override fun onDestroyView() {
+        super.onDestroyView()
+        _binding = null
+    }
```

Comment:
```suggestion
    override fun onDestroyView() {
        _binding = null
        super.onDestroyView()
    }
```

이쪽이 안정성이 더 좋습니다!! 이유는 뭘까요?!